<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralStatusBar</class>
 <widget class="QWidget" name="GeneralStatusBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="librariesCount">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0 Librerie</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="openedPackages">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Pacchetti aperti:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="packagePath">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Percorso pacchetto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Sfoglia</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="problemsCounter">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0 Problemi</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
